### PR TITLE
Switch entire backend to Node!

### PR DESCRIPTION
**Pull request by dannyradden**

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+);
+app.use(passport.initialize());
+app.use(passport.session());
+
+require('./routes/authRoutes')(app);
+require('./routes/campaignRoutes')(app);
+require('./routes/signatureRoutes')(app);
+
+const path = require('path');
+console.log('aaaaaaaaaaaa', path.resolve(__dirname, '../client', 'build', 'index.html'));
```
Comment (wdoug) [inline backend/index.js]: ✂️

File: backend/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+require('./services/passport');
+
+mongoose.connect(keys.mongoURI);
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(
+  cookieSession({
+    maxAge: 30 * 24 * 60 * 60 * 1000,
```
Comment (chompasina) [inline backend/index.js]: What does this maxAge do exactly?

File: client/src/registerServiceWorker.js [hunk 3]
```diff
@@ -8,43 +8,96 @@
...
+    });
+}
+
 export function unregister() {
   if ('serviceWorker' in navigator) {
-    navigator.serviceWorker.ready.then(registration => {
+    navigator.serviceWorker.ready.then((registration) => {
       registration.unregister();
     });
   }
```
Comment (chompasina) [inline client/src/registerServiceWorker.js]: Did you say we are using this service worker?

File: client/src/routes.js [hunk 4]
```diff
@@ -64,20 +56,6 @@ export default (
...
-      status={404}
-      getComponent={(location, callback) => callback(null, NotFound)}
-    />
-    {/* Catch all route */}
-    <Route
-      path="*"
-      status={404}
-      onEnter={(nextState, replace) => replace('/error')}
-    />
+    <Route path="/about" getComponent={(location, callback) => callback(null, About)} />
```
Comment (chompasina) [inline client/src/routes.js]: I think we'll want the 404 /error and catch all route still, or is there somewhere else they're included?

**Pull request merged**